sdk/OoyalaSkinSDK/index.android.js

```diff
@@ -70,7 +80,12 @@ var OoyalaSkin = React.createClass({
   },
 
   onPress: function() {
-    eventBridge.onPress({"name":"Play"});
+    eventBridge.onPress({"name":"play"});
```

Comment:
What is this onPress used for?  If it is useful, we need to provide a valid "name" value (like PlayPause)